l10n/template.ftl

```diff
@@ -0,0 +1,288 @@
+# WARNING: do not edit this file, it's automatically generated by ExtractL10nPlugin.
+# If you need to manually add strings, do so in ./locales/en-US.ftl. See ./README.md for more details.
+
+article-footer-last-modified = This page was last modified on <time data-l10n-name="date">{ $date }</time> by <a data-l10n-name="contributors">MDN contributors</a>.
+article-footer-source-title = Folder: { $folder } (Opens in a new tab)
+baseline-asterisk = Some parts of this feature may have varying levels of support.
+baseline-high-extra = This feature is well established and works across many devices and browser versions. It’s been available across browsers since { $date }.
+baseline-low-extra = Since { $date }, this feature works across the latest devices and browser versions. This feature might not work in older devices or browsers.
+baseline-not-extra = This feature is not Baseline because it does not work in some of the most widely-used browsers.
+baseline-supported-in = Supported in { $browsers }
+baseline-unsupported-in = Not widely supported in { $browsers }
+baseline-supported-and-unsupported-in = Supported in { $supported }, but not widely supported in { $unsupported }
+homepage-hero-title = Resources for Developers,<br> by Developers
+homepage-hero-description = Documenting <a data-l10n-name="css">CSS</a>, <a data-l10n-name="html">HTML</a>, and <a data-l10n-name="js">JavaScript</a>, since 2005.
+not-found-title = Page not found
+not-found-description = Sorry, the page <code data-l10n-name="url">{ $url }</code> could not be found.
+not-found-fallback-english = <strong data-l10n-name="strong">Good news:</strong> The page you requested exists in <em data-l10n-name="em">English</em>.
+not-found-fallback-search = The page you requested doesn't exist, but you could try a site search for:
+not-found-back = Go back to the home page
+reference-toc-header = In this article
+footer-mofo = Visit <a data-l10n-name="moco">Mozilla Corporation’s</a> not-for-profit parent, the <a data-l10n-name="mofo">Mozilla Foundation</a>.
+footer-copyright = Portions of this content are ©1998–{ $year } by individual mozilla.org contributors. Content available under <a data-l10n-name="cc">a Creative Commons license</a>.
+search-modal-site-search = Site search for <em>{ $query }</em>
+site-search-search-stats = Found { $results } documents.
+site-search-suggestion-matches =
```

Comment:
While this is technically valid fluent it's best to avoid this pattern. This should be two strings - otherwise you can't guarantee that all patterns will have translations ([reference](https://github.com/projectfluent/fluent/wiki/Good-Practices-for-Developers#prefer-separate-messages-over-variants-for-ui-logic)). 

Typically you want to remove the logic from the strings themselves, so it's preferable to have a `-greaterthan` and a `-equals` string then have the logic happening outside of fluent to choose the correct string.